types/assert-equal-jsx/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for assert-equal-jsx 1.0
+// Project: https://github.com/thejameskyle/assert-equal-jsx
+// Definitions by: Josh Toft <https://pack.io>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as React from 'react';
+
+export type SanitizeFunc = (str: string) => string;
```

Comment:
That lint rule tells you to prefer `sanitize(str: string): string;` over `sanitize: (str: string) => string;`.